scripts/update_showcase.js

```diff
@@ -7,10 +7,10 @@ Script for updating Showcase Template App from the SDK Example app.
 
 const yargs = require("yargs")
```

Comment:
If you run this script multiple times, it adds the dependencies to the projects multiple times, thus breaking the compilation.

Reply:
This is fixed now. Please review again.